src/xx-apt

```diff
@@ -139,6 +139,15 @@ if [ "$TARGETARCH" = "riscv64" ] && [ "$(xx-info vendor)" = "debian" ]; then
   apt-get install -y debian-ports-archive-keyring
   echo "deb [ arch=riscv64 ] http://ftp.ports.debian.org/debian-ports sid main" >>/etc/apt/sources.list.d/riscv64-sid.list
 fi
+if [ "$TARGETARCH" = "loongarch64" ] && ( [ "$(xx-info vendor)" = "debian" ] || [ "$(xx-info vendor)" = "Loongnix" ] ); then
+  apt-get update
+  apt-get install -y debian-ports-archive-keyring
+  if [ -z "$XX_APT_REPO_URL_LOONGARCH64" ]; then
+  echo "Please set the environment variable XX_APT_REPO_URL_LOONGARCH64=pkg.loongnix.cn"
```

Comment:
http://ftp.ports.debian.org/debian-ports/pool-loong64/ does not work as a default?

Reply:
> http://ftp.ports.debian.org/debian-ports/pool-loong64/ does not work as a default?

At present, this link does not work. Can it be for the time being, and then correct it after actual use?

Reply:
What's the issue with the debian link?

Reply:
> What's the issue with the debian link?

Since the ABI atmosphere ABI 1.0 and ABI2.0 under the ABI atmosphere under Loongarch, the current debian official source link supports ABI2.0. The source link in the current code is the ABI 1.0 software source maintained by Loongarch, ABI1.0 and ABI2.0 The distinction can be performed through Loongarch64 and Loong64. Thank you for your reminder!